### PR TITLE
Increase test coverage for message.py, user.py, views.py

### LINKED ISSUE #49: Increase coverage for message.py, user.py and views.py
Missing lines:
* /bottery/message.py: 20
* /bottery/user.py: 4-8
* /bottery/views.py: 1-2

*See Travis log to confirm the missing lines*: https://travis-ci.org/rougeth/bottery

### PR DESCRIPTION
**Pull request by gabrielgradinaru**
Fixes #49

### CODE REVIEW

**Comment by leportella** [comment]
@gabrielgradinaru I think this need a rebase!

File: tests/test_message.py [hunk 1]
```diff
@@ -12,6 +13,8 @@ def test_render(mocked_settings):
     specified context.
     '''
     message = Message(123, 'Telegram', 'Bottery', 'text', 1506191745, 'raw')
+    assert message.datetime == datetime.datetime(2017, 9, 23, 18, 35, 45)
+
```
Comment (rougeth) [inline tests/test_message.py]: Shouldn't we have a specific test for the `Message.datetime`?

File: tests/test_cli.py [hunk 2]
```diff
@@ -47,4 +47,4 @@ def test_startproject():
 
         assert result.exit_code == 0
         assert os.listdir() == [project_name]
-        assert os.listdir(project_name) == project_files
+        assert os.listdir(project_name).sort() == project_files.sort()
```
Comment (nicoddemus) [inline tests/test_cli.py]: `.sort()` returns `None`, so you are comparing `None == None`. 😉 

This should be:

```python
assert sorted(os.listdir(project_name)) == sorted(project_files)
```

File: tests/test_views.py [hunk 3]
```diff
@@ -0,0 +1,5 @@
+from bottery.views import ping
+
+assert ping('any_string') == 'pong'
```
Comment (nicoddemus) [inline tests/test_views.py]: This should be inside a test function:

```python
def test_ping_pong():
    assert ping('any_string') == 'pong'
    ...
```

**Comment by nicoddemus** [comment]
Perhaps we should consider using [`attrs`](http://www.attrs.org/en/stable/) instead of coding classes like `Message`, `User`, etc? This would make for correct classes without boilerplate and no need to test their basic functionality. Here's a nice post about it that makes a good read: [The One Python Library Everyone Needs](https://glyph.twistedmatrix.com/2016/08/attrs.html)

**Comment by gabrielgradinaru** [comment]
@nicoddemus This was an interesting read indeed and it is the first time I have heard about this module. It looks interesting and I could make another pull request adding it if it's decided we should use it.

File: tests/test_message.py (hunk 1 above)
Comment (gabrielgradinaru) [reply tests/test_message.py]: done

File: tests/test_cli.py (hunk 2 above)
Comment (gabrielgradinaru) [reply tests/test_cli.py]: done

File: tests/test_views.py (hunk 3 above)
Comment (gabrielgradinaru) [reply tests/test_views.py]: done

**Comment by rougeth** [comment]
I would like to have the minimal dependencies as possible but obviously without reinventing the wheels. I don't know `attrs`, but I think we should wait to see what this classes will really be.

**Pull request merged**